doc/api/deprecations.md

```diff
@@ -2518,6 +2518,27 @@ Type: Documentation-only
 Prefer [`response.socket`][] over [`response.connection`] and
 [`request.socket`][] over [`request.connection`].
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: http finished
```

Comment:
The deprecation seems to be valid for `http` and `http2` and the property is available on the request and the response. Maybe the following is clearer?

```suggestion
### DEP0XXX: http/2 request.finished and response.finished
```

Reply:
Where do you see it on the request?

Reply:
https://github.com/nodejs/node/pull/28679/files#diff-69e1ac0b5bfc06e74f2c1ab7b062c6afR606